opentelemetry-tracing/README.adoc

```diff
@@ -0,0 +1,213 @@
+otel-collector_1  | 2023-12-13T21:05:28.002Z    info    TracesExporter  {"kind": "exporter", "data_type": "traces", "name": "logging", "resource spans": 1, "spans": 1}
+otel-collector_1  | 2023-12-13T21:05:28.002Z    info    ResourceSpans #0
+otel-collector_1  | Resource SchemaURL: https://opentelemetry.io/schemas/1.20.0
+otel-collector_1  | Resource attributes:
+otel-collector_1  |      -> service.name: Str(opentelemetry-tracing.war)
+otel-collector_1  |      -> telemetry.sdk.language: Str(java)
+otel-collector_1  |      -> telemetry.sdk.name: Str(opentelemetry)
+otel-collector_1  |      -> telemetry.sdk.version: Str(1.29.0)
+otel-collector_1  | ScopeSpans #0
+otel-collector_1  | ScopeSpans SchemaURL:
+otel-collector_1  | InstrumentationScope io.smallrye.opentelemetry 2.6.0
+otel-collector_1  | Span #0
+otel-collector_1  |     Trace ID       : c761e8fadec36d222adac36dcff1f4b1
+otel-collector_1  |     Parent ID      :
+otel-collector_1  |     ID             : 08f93dd25f75b5cd
+otel-collector_1  |     Name           : GET /opentelemetry-tracing/implicit-trace
+otel-collector_1  |     Kind           : Server
+otel-collector_1  |     Start time     : 2023-12-13 21:05:20.560054393 +0000 UTC
+otel-collector_1  |     End time       : 2023-12-13 21:05:20.621635685 +0000 UTC
+otel-collector_1  |     Status code    : Unset
+otel-collector_1  |     Status message :
+otel-collector_1  | Attributes:
+otel-collector_1  |      -> net.host.port: Int(8080)
+otel-collector_1  |      -> http.scheme: Str(http)
+otel-collector_1  |      -> http.method: Str(GET)
+otel-collector_1  |      -> http.status_code: Int(200)
+otel-collector_1  |      -> net.transport: Str(ip_tcp)
+otel-collector_1  |      -> user_agent.original: Str(curl/8.2.1)
+otel-collector_1  |      -> net.host.name: Str(localhost)
+otel-collector_1  |      -> http.route: Str(/opentelemetry-tracing/implicit-trace)
+otel-collector_1  |      -> http.target: Str(/opentelemetry-tracing/implicit-trace)
+otel-collector_1  |      -> net.sock.host.addr: Str(127.0.0.1)
+otel-collector_1  |     {"kind": "exporter", "data_type": "traces", "name": "logging"}
+-----
+
+// Server Distribution Testing
+include::../shared-doc/run-integration-tests-with-server-distribution.adoc[leveloffset=+2]
+// Undeploy the Quickstart
+include::../shared-doc/undeploy-the-quickstart.adoc[leveloffset=+2]
+//  Restore the {productName} Standalone Server Configuration
+include::../shared-doc/restore-standalone-server-configuration.adoc[leveloffset=+2]
+// Restore the {productName} Standalone Server Configuration Manually
+include::../shared-doc/restore-standalone-server-configuration-manual.adoc[leveloffset=+3]
+// Build and run sections for other environments/builds
+ifndef::ProductRelease,EAPXPRelease[]
+include::../shared-doc/build-and-run-the-quickstart-with-provisioned-server.adoc[leveloffset=+1]
+endif::[]
+// Bootable JAR
+include::../shared-doc/build-and-run-the-quickstart-with-bootable-jar.adoc[leveloffset=+1]
+// OpenShift
```

Comment:
We should also provide instructions on how to setup the docker service for openshift.